source/technical/ep.rst

```diff
@@ -0,0 +1,435 @@
+.. _experiment-precis:
+
+==============================
+Experiment Precis
+==============================
+
+.. _ep-introduction:
+
+Introduction
+____________
+
+Chameleon records experiment setup (OpenStack) events that users performed on the testbed, such as creating leases, creating instances, and setting up networks. 
+Users can request their experiment records from Chameleon using their Chameleon credentials. A report on those experiment records is known as *Experiment Precis*. 
+An *Experiment Precis* is bounded to a lease. Chameleon defines an *experiment* as a series of testbed setup (OpenStack) events a user performed under a lease of a project. 
+Using *Experiment Precis*, users will be able to analyze, understand and even replay their experiments. 
+
+.. warning::
+
+   The CEP service is **only** available at `CHI@UC <https://chi.uc.chameleoncloud.org>`_. 
```

Comment:
I wouldn't use the CEP abbreviation here yet, unless you introduce it above like "The Chameleon Experiment Precis (CEP)..."

```suggestion
   The *Experiment Precis* service is **only** available at `CHI@UC <https://chi.uc.chameleoncloud.org>`_. 
```